documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -196,6 +216,10 @@ This option will expand your installation to fill the whole SD card, giving you
 
 WARNING: There is no confirmation: selecting the option begins the partition expansion immediately.
 
+===== Compositor
+
+Enable or disable the xcompmgr composition manager.
```

Comment:
I was told that this option was going to be removed, which is why I didn't removed it in https://github.com/raspberrypi/documentation/commit/48d6a70e52c9f180ee4b721d7d0051721416c840 :shrug:

Reply:
🤦‍♂️

Reply:
Feels likely. 

@jehlers42 why did this get added back in? Do you have mail from Simon about this? If not, please remove this section in another PR.

Reply:
I added it back in since it was reflected in the screenshot I captured. Will do another check with the most up-to-date nightly and remove it again if it's not there.

Reply:
May be a not-a-Pi4 thing?

Reply:
Okay it's now gone again in the newest nightly. PR out: https://github.com/raspberrypi/documentation/pull/3114